### PR TITLE
Vale errors and suggestions

### PR DESCRIPTION
**Pull request by HeatherStenson**
Working on files in the `/getting-started/` folder:

Before: `✖ 55 errors, 189 warnings and 370 suggestions in 14 files.`
After: `✖ 13 errors, 132 warnings and 307 suggestions in 14 files.`

### CODE REVIEW

File: src/pages/docs/getting-started/creating-the-first-collection.md [hunk 1]
```diff
@@ -35,25 +35,25 @@ contextual_links:
...
 ---
 
-Postman Collections are a group of saved requests. Every request you send in Postman appears under the **History** tab of the sidebar. On a small scale, reusing requests through the history section is convenient. However, as your Postman usage grows, it can be time-consuming to find a particular request in your history. Instead of scrolling through your history section, you can save all your requests as a group for easier access.
-
-Let's review how to [send a basic request](/docs/getting-started/sending-the-first-request/) and create a new collection.
+Postman Collections are a group of saved requests. Every request you send in Postman appears under the **History** tab of the sidebar. On a small scale, reusing requests through the history section is convenient. As your Postman usage grows, it can be time-consuming to find a particular request in your history. Instead of scrolling through your history section, you can save all your requests as a group for easier access.
 
 ## Creating your first collection
 
+To create a new collection by first creating a new request:
```
Comment (claireriley) [inline src/pages/docs/getting-started/creating-the-first-collection.md:42]: This reads a bit awkwardly to me, maybe try something like

"To create a new collection, you must first create a request:"

File: src/pages/docs/getting-started/creating-your-first-workspace.md [hunk 2]
```diff
@@ -35,8 +35,8 @@ To create a new workspace:
 
 1. Enter a **Name** and optional **Summary** for your workspace.
 1. Choose the workspace's visibility:
-    * A **Personal** workspace is only visible to you.
-    * A **Private** workspace is only visible to you and to any team members you invite to join it.
+    * A **Personal** workspace is visible to you.
```
Comment (claireriley) [inline src/pages/docs/getting-started/creating-your-first-workspace.md:38]: I think we should keep "only" in this line, to me it comes across as more clear and direct.

In ComplexWords.yml it offers "only" as a preferable alternative, although Weasel.yml doesn't like it...not sure which way we want to go but something to consider

File: src/pages/docs/getting-started/importing-and-exporting-data.md [hunk 3]
```diff
@@ -106,10 +106,10 @@ You can convert your collection's format from v1 to v2 to import it into Postman
 1. Convert an individual Postman collection from v1 to v2 by entering the command below.
 
    ```bash
-   postman-collection-transformer convert -i <path to input Postman collection file> -o <path where the output Postman file will be downloaded> -j 1.0.0 -p 2.0.0 -P
+   postman-collection-transformer convert -i <path to input Postman collection file> -o <path where the output Postman file download> -j 1.0.0 -p 2.0.0 -P
```
Comment (claireriley) [inline src/pages/docs/getting-started/importing-and-exporting-data.md:109]: Would "will download" work here? Just "download" reads unclearly to me

File: src/pages/docs/getting-started/importing-and-exporting-data.md [hunk 4]
```diff
@@ -182,13 +182,13 @@ You can import data in bulk from a Bitbucket repository.
 
 1. In Postman, select your Bitbucket workspace, repository, and branch, then select **Continue**.
 
-1. A preview of the files that will be imported to your workspace is displayed. Select **Import** to confirm and complete the import.
+1. Postman shows a preview of the files to import to your workspace. Select **Import** to confirm and complete the import.
```
Comment (claireriley) [inline src/pages/docs/getting-started/importing-and-exporting-data.md:185]:
```suggestion
1. Postman shows a preview of the files to import into your workspace. Select **Import** to confirm and complete the import.
```

File: src/pages/docs/getting-started/postman-account.md [hunk 5]
```diff
@@ -71,13 +71,13 @@ During the sign up process, you have the option to create or join a [team](/docs
...
 
 > You can opt out of the sign in process at any time by navigating back to the Postman app and selecting **Skip and take me to Postman Desktop App**.
 
 Sign into Postman in your browser by entering your account credentials or signing in with Google. If you are on a Postman Professional or Enterprise plan, opt to **Sign in with Single Sign-On (SSO)**.
 
-Check **Keep me signed in** if you would like to remain signed in after your current session for 30 minutes before re-authenticating. If you do not want to remain signed in on the computer you are working on, uncheck this option. Note that you will be prompted to sign in again after 30 minutes.
+Check **Keep me signed in** if you would like to remain signed in after your current session for 30 minutes before re-authenticating. If you don't want to remain signed in on the computer you are working on, turn this option off. Note that you will be prompted to sign in again after 30 minutes.
```
Comment (claireriley) [inline src/pages/docs/getting-started/postman-account.md:80]: "Uncheck" reads more explicitly than "turn off", might be one to consider keeping for now and/or discussing in a style guide sync

File: src/pages/docs/getting-started/syncing.md [hunk 6]
```diff
@@ -26,21 +26,21 @@ warning: false
 
 ---
 
-Postman automatically syncs changes across all devices that are linked to your account, including edits, updates, additions, or deletions. As long as you are signed in to Postman in the desktop app or in the web version, your data will be synced.
+Postman automatically syncs changes across all devices that are linked to your account, including edits, updates, additions, or deletions. As long as you are signed in to Postman in the desktop app or in the web version, your data syncs.
```
Comment (claireriley) [inline src/pages/docs/getting-started/syncing.md:29]: possible alternative for clarity:
```suggestion
Postman automatically syncs changes across all devices that are linked to your account, including edits, updates, additions, or deletions. As long as you are signed into Postman in the desktop app or in the web version, and have a reliable internet connection, your data will sync.
```

**Review by claireriley — APPROVED** [review]
Looks good! Noted a few minor suggestions

File: src/pages/docs/getting-started/creating-the-first-collection.md [hunk 7]
```diff
@@ -35,25 +35,25 @@ contextual_links:
...
+
 You can create a new request from the left sidebar in Postman. Select __New__ &gt; __HTTP Request__.
 
 <img alt="Create new request sidebar" src="https://assets.postman.com/postman-docs/create-new-request-sidebar-v9.jpg"/>
 
-Enter a request in the request builder and select **Save** to open the **SAVE REQUEST** modal.
+Enter a request in the request builder and select **Save** to open the **Save Request** modal.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/creating-the-first-collection.md:48]:
```suggestion
Enter a request in the request builder and select **Save** to open the **Save Request** dialog box.
```

Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/creating-the-first-collection.md:48]: Or you can use "window" but I think we're trying to avoid "modal"

File: src/pages/docs/getting-started/creating-the-first-collection.md [hunk 8]
```diff
@@ -35,25 +35,25 @@ contextual_links:
...
 
 1. Create a new collection by selecting **New Collection**. Enter a collection name, and then select the **Create** button.
 
     <img alt="Save request to collection" src="https://assets.postman.com/postman-docs/save-request-to-collection-v9.jpg" width="400px"/>
 
 1. Select **Save** to save the request in the new collection.
 
-Congratulations! You can see all of your collections under the **Collections** tab of the sidebar on the left, as illustrated below:
+Congratulations! You can see all your collections under the **Collections** tab of the sidebar on the left, as illustrated below:
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/creating-the-first-collection.md:56]:
```suggestion
Congratulations! You can see all your collections under the **Collections** tab of the sidebar on the left, as shown below:
```

Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/creating-the-first-collection.md:56]: Seriously vale "illustrated" isn't too wordy but "monitor" is?? 😄

File: src/pages/docs/getting-started/creating-your-first-workspace.md [hunk 9]
```diff
@@ -23,9 +23,9 @@ contextual_links:
     url: "/docs/collaborating-in-postman/using-workspaces/creating-workspaces/"
 ---
 
-Postman workspaces allow you to organize and collaborate on API projects with your team. Within each workspace you can share [APIs](/docs/designing-and-developing-your-api/creating-an-api/), [collections](/docs/getting-started/creating-the-first-collection/), [environments](/docs/sending-requests/managing-environments/), and other components of your work in Postman.
+Postman workspaces allow you to organize and work together on API projects with your team. Within each workspace you can share [APIs](/docs/designing-and-developing-your-api/creating-an-api/), [collections](/docs/getting-started/creating-the-first-collection/), [environments](/docs/sending-requests/managing-environments/), and other components of your work in Postman.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/creating-your-first-workspace.md:26]:
```suggestion
Postman workspaces enable you to organize and work together on API projects with your team. Within each workspace you can share [APIs](/docs/designing-and-developing-your-api/creating-an-api/), [collections](/docs/getting-started/creating-the-first-collection/), [environments](/docs/sending-requests/managing-environments/), and other components of your work in Postman.
```

File: src/pages/docs/getting-started/exploring-public-api-network.md [hunk 10]
```diff
@@ -41,7 +41,7 @@ There are a variety of ways to find APIs, workspaces, teams, and collections:
 * **Categories** showcases popular API categories in the Public API Network including Communication, Data Analytics, and Financial Services.
 * **In the spotlight** highlights the latest exciting, useful, and noteworthy contributions to the Public API Network, handpicked by the experts at Postman.
 * **Trending workspaces, collections, and APIs** shows entities that are the most popular in the Public API Network, based on the number of people viewing or creating forks of them.
-* The **Search Postman** box at the top of the page searches all of Postman by default, but you can also use the search controls to limit a search to only the Public API Network.
+* The **Search Postman** box at the top of the page searches the entirety of Postman by default, but you can also use the search controls to limit a search to the Public API Network.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/exploring-public-api-network.md:44]: personally I prefer "all" to "the entirety" 🙂

File: src/pages/docs/getting-started/installation-and-updates.md [hunk 11]
```diff
@@ -169,22 +169,22 @@ The native Postman apps will notify you when a major update is available. For ot
...
+Select the update option to download or install the latest update. You will see a notification when the download is complete, prompting you to restart the Postman app to apply the updates. If you don't want to update yet, choose __Later__ to auto-update the next time you launch the app.
 
 You can configure your preferences to enable automatic download for major updates in __Settings__ &gt; __Update__. Postman automatically downloads minor updates and bug fixes.
 
 <img alt="Check for updates" src="https://assets.postman.com/postman-docs/settings-updates-v9.jpg"/>
 
 ## Postman Chrome app (deprecated)
 
-The Postman Chrome app is deprecated—if you're using the Chrome app, you can [retain your data when you switch to the native app](#migrating-to-the-native-app) either by syncing with a Postman account you're signed into, or by exporting from Chrome and importing into the native app.
+The Postman Chrome app is deprecated. If you use the Chrome app, you can [retain your data when you switch to the native app](#migrating-to-the-native-app) either by syncing with a signed-in Postman account or by exporting from Chrome and importing into the native app.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/installation-and-updates.md:180]: I feel like "signed-in" could just be deleted

File: src/pages/docs/getting-started/postman-account.md (hunk 5 above)
Comment (mark-anthony-postman) [reply src/pages/docs/getting-started/postman-account.md:80]: Per Microsoft you either select or clear a check box.
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/check-checkbox-check-mark

**Review by mark-anthony-postman — APPROVED** [review]
In general LGTM! 🌈  Just don't tell vale I disagree with it in a few places 😉

File: src/pages/docs/getting-started/creating-the-first-collection.md (hunk 8 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/creating-the-first-collection.md:56]: Lerl, I didn't even notice this 😂

File: src/pages/docs/getting-started/creating-your-first-workspace.md (hunk 9 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/creating-your-first-workspace.md:26]: Good replacement! I will be on the lookout for instances of "allow" that would benefit from being "enable" instead 🔍

File: src/pages/docs/getting-started/postman-account.md (hunk 5 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/postman-account.md:80]: Good call @claireriley and @mark-anthony-postman - I will reword to use "clear" instead.

File: src/pages/docs/getting-started/syncing.md (hunk 6 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/syncing.md:29]: Much better, thank you!

File: src/pages/docs/getting-started/creating-the-first-collection.md (hunk 1 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/creating-the-first-collection.md:42]: Good call! Rewording

File: src/pages/docs/getting-started/exploring-public-api-network.md (hunk 10 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/exploring-public-api-network.md:44]: Haaaa, yes. I'll reword!

File: src/pages/docs/getting-started/importing-and-exporting-data.md (hunk 3 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/importing-and-exporting-data.md:109]: Oops, thanks for catching this! I meant to reword this.

File: src/pages/docs/getting-started/installation-and-updates.md (hunk 11 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/installation-and-updates.md:180]: Good call!
